xacro/__init__.py

```diff
@@ -138,15 +143,19 @@ def __str__(self):
 
 verbosity = 1
 # deprecate non-namespaced use of xacro tags (issues #41, #59, #60)
+
+
 def deprecated_tag(_issued=[False]):
     if _issued[0]:
         return
     _issued[0] = True
 
     if verbosity > 0:
-        warning("deprecated: xacro tags should be prepended with 'xacro' xml namespace.")
+        warning(
+            "deprecated: xacro tags should be prepended with 'xacro' xml namespace.")
         message("""Use the following script to fix incorrect usage:
-        find . -iname "*.xacro" | xargs sed -i 's#<\([/]\\?\)\(if\|unless\|include\|arg\|property\|macro\|insert_block\)#<\\1xacro:\\2#g'""")
+        find . -iname "*.xacro" | xargs sed -i 's#<\\([/]\\?\\)\\(if\\|unless\\|include\\|arg
```

Comment:
Hm, this seems weird.  Why are extra slashes being added to the regex here?  Also, I know this was probably split because of one of the linters, but I think the split makes it much harder to read.  I'd rather disable the linting for this line instead.